src/modal/modal.spec.ts

```diff
@@ -882,6 +893,16 @@ export class WithAutofocusModalCmpt {
 export class WithFirstFocusableModalCmpt {
 }
 
+@Component({
+  selector: 'modal-skip-tabindex-firstfocusable-cmpt',
+  template: `
+  <button tabindex="-1" class="firstFocusable close">Close</button>
+  <button class="other">Other button</button>
+`
+})
+export class WithSkipTabindexFirstFocusableModalCmpt {
```

Comment:
Good naming! Though, could you change to `WithNegativeTabIndexElementThatShouldBeSkippedWhileFindingFirstFocusableElementModalComponent`? 

Just kidding :)